lib/push.js

```diff
@@ -178,29 +178,30 @@ module.exports = function (log, db, config) {
    * if they receive an unsupported message type.  Filter out
    * devices that we know won't respond well to the given command.
    *
-   * @param {String} command
-   * The command from the push message payload
+   * @param {Object} data
```

Comment:
naming nit: can we call this "payload" or something, to avoid having "data.data.reason" below?